README.md

```diff
@@ -1850,6 +1852,76 @@ Server: spray-can/1.3.3
 }
 ```
 
+## POST /api/workflows/:version/:id/call-caching
+
+This endpoint allows for reconfiguration of call cache result reuse settings for all calls within a workflow.
+
+Accepted parameters are:
+
+<ul>
+  <li>`allow` Mandatory boolean value, specifies whether call cache result reuse is allowed for all calls in the
+   specified workflow.</li>
+</ul>
```

Comment:
huh I didn't know you could use `<ul>` in markdown, I've always done

* item 1
* item2